tests/test_deploy_content.py

```diff
@@ -82,7 +82,7 @@ def test_create_or_return_space_none_found(mocker):
     sdk.create_space.return_value = models.Space(name="Foo", parent_id="1", id="42")
     deploy_content.get_space_ids_from_name.return_value = []
 
-    target_id = deploy_content.create_or_return_space("foo", "2", sdk)
+    target_id = deploy_content.create_or_return_space("foo", "5", sdk)
```

Comment:
I updated this to be a number other than 1-4.  The User folder is typically 2, and there is now an error message that prints noting that folders can't be created in the Users folder via API, so this pytest was failing.  Put in 5, as 1-4 are typically Shared/Users/Embed Users/Embed Groups